src/plugin.ts

```diff
@@ -76,18 +82,22 @@ export function openGraphImagePlugin(options: Options): Plugin {
   const appUrlPromise = new DeferredPromise<URL>()
   const routesWithImages = new Set<ConfigRoute>()
 
-  async function fromRoot(...paths: Array<string>): Promise<string> {
-    const viteConfig = await viteConfigPromise
-    return path.resolve(viteConfig.root, ...paths)
-  }
-
   async function fromRemixApp(...paths: Array<string>): Promise<string> {
     const remixContext = await remixContextPromise
     return path.resolve(remixContext.remixConfig.appDirectory, ...paths)
   }
 
+  async function fromRemixAssets(...paths: Array<string>): Promise<string> {
+    const remixContext = await remixContextPromise
+    return path.resolve(
+      remixContext.remixConfig.assetsBuildDirectory,
```

Comment:
is the `assetsBuildDirectory` always defined? it was coming as `undefined` when i had tried.

Reply:
You're right, it's not. I changed this to point to `buildDirectory`, which is always defined. That required me to update the types for `remixConfig` to use the Vite plugin options, not the resolved config type.